src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/InnvilgetSkolepenger/SkoleårDelårsperiode.tsx

```diff
@@ -77,6 +84,8 @@ const SkoleårDelårsperiode: React.FC<ValideringsPropsMedOppdatering<IPeriodeSk
         }
     };
 
+    const erLesevisning: boolean = !behandlingErRedigerbar || !!skoleårErFjernet;
```

Comment:
For innvilgelse - vil ikke denne alltid bli true?

Reply:
Koden er nok riktig, men lettere å lese 
`!behandlingenErRedigerbar || skoleårErFjernet == true`